assets/app/views/settings.html

```diff
@@ -8,16 +8,16 @@
       <h2>General information</h2>
       <!-- TODO make these fields editable - probably with a cog icon that flips into inline-editing -->
       <dl class="dl-horizontal left">
-        <dt>Name:</dt>
-        <dd>{{projectName}}</dd>
-        <dt>Display name:</dt>
-        <dd ng-if="project | displayName: true">{{project | displayName: true}}</dd>
-        <dd ng-if="!(project | displayName: true)"><em>No display name</em></dd>
-        <dt>Description:</dt>
-        <dd ng-if="project | description">
+        <dt class="name">Name:</dt>
```

Comment:
not a huge fan of classes for location... I'd almost rather make these ids, but I'm not sure how to do that best. mostly, I don't want to accidentally pick up appearance styles from generic class names